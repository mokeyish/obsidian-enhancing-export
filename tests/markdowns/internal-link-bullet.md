- potato
- [[#more complex vegetables]]
- cucumber
