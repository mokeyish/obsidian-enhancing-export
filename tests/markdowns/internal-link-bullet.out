Pandoc
  Meta { unMeta = fromList [] }
  [ BulletList
      [ [ Plain [ Str "potato" ] ]
      , [ Plain
            [ Link
                ( "" , [] , [] )
                [ Str "more"
                , Space
                , Str "complex"
                , Space
                , Str "vegetables"
                ]
                ( "#more-complex-vegetables" , "" )
            ]
        ]
      , [ Plain [ Str "cucumber" ] ]
      ]
  ]
